### PR TITLE
chore: remove useless swc_core features

### PR DESCRIPTION
**Pull request by JSerFeng**
## Summary

Remove some useless features of swc_core

## Checklist

- [ ] Tests updated (or not required).
- [ ] Documentation updated (or not required).

### CODE REVIEW

File: Cargo.lock
Comment (chenjiahan) [inline Cargo.lock:1]: maybe this change can be marked as `perf` instead of `chore`? It should reduce the binary size.

**Comment by h-a-n-a** [comment]
Looks good

File: Cargo.lock
Comment (JSerFeng) [reply Cargo.lock:1]: I tried diff the size using cargo-float but see no differences, these features may already be ignored by the compiler I suppose

Comment (chenjiahan) [reply Cargo.lock:1]: get

**Review by chenjiahan — APPROVED** [review]

**Pull request merged**